### PR TITLE
Feat/ccn3 2012/update dbs content design

### PR DESCRIPTION
**Pull request by jenniferbrown94**
## Description

[Goes here]

## Todo's before PR

- [ ] Branch has been rebased against develop
- [ ] Unit tests passed (`make test`)
- [ ] Selenium tests have passed
- [ ] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [ ] PR description describes the overall goals of PR commits
- [ ] Templates have been checked against the relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [ ] Specified reviewers (even if it's yourself)
- [ ] Specified assignees (those who'll merge)
- [ ] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

**Comment by Thomas-95IS** [comment]
rerun jenkins build

**Comment by Thomas-95IS** [comment]
This will be able to go in pending approval and changes to https://github.com/InformedSolutions/OFS-MORE-Nanny-Gateway/pull/

File: application/tests/test_dbs/tests_routing.py [hunk 1]
```diff
@@ -194,116 +173,114 @@ def test_yes_to_dbs_update_serice_page_redirects_to_non_capita_dbs_details_page(
...
-        self.assertTrue(patch_mock.called)
-        patch_mock.assert_called_once_with('application', params={'application_id': self.app_id, 'dbs_status': 'COMPLETED'})
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(resolve(response.url).func.__name__, TaskListView.__name__)
 
+    def test_dbs_update_check_redirect_post_dbs(self, *args):
+        response = self.client.get(reverse('dbs:DBS-Update-Check-View') + self.url_suffix)
 
-class CriminalRecordFormsTest(SimpleTestCase):
-    def test_less_than_12_digit_dbs_raises_error(self):
```
Comment (Thomas-95IS) [inline application/tests/test_dbs/tests_routing.py]: Have all the form-level tests been axed for a reason?

Comment (jenniferbrown94) [reply application/tests/test_dbs/tests_routing.py]: they are in a separate file called test_form_validation

**Pull request merged**